### ISSUE TITLE
Loss decrease unstable when training TOMP

### ISSUE THREAD

**Issue by chenshihfang** [visionml/pytracking#362]
HI,
when I use the default setting for training TOMP (tomp50),
the training loss curves decreased unstably in the beginning,
what may be the reason to effect this and how to deal with it?
![image](https://user-images.githubusercontent.com/113094161/202708516-51054ed3-dddb-4008-8bb5-02b04121fdaf.png)

**Comment by 2006pmach** [comment]
Hey, yes we also observed that sometimes. Adding gradient norm clipping can help to overcome this. https://pytorch.org/docs/stable/generated/torch.nn.utils.clip_grad_norm_.html
